packages/server/datasource-sha.env

```diff
@@ -1,4 +1,4 @@
-MSSQL_SHA=sha256:c4369c38385eba011c10906dc8892425831275bb035d5ce69656da8e29de50d8
+MSSQL_SHA=sha256:3b913841850a4d57fcfcb798be06acc88ea0f2acc5418bc0c140a43e91c4a545
```

Comment:
Why this update? Could we add a comment on what is the actual version of the sha?

Reply:
This was reducing our MS-SQL version as the latest versions have JSON aggregation functions, while older versions use a different syntax - we want to test against the minimum version we would want to support.